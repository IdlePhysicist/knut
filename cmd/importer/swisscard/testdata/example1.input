Transaction Date, Posting Date, Card Number ,Billing Amount, Description, Merchant City , Merchant State , Merchant Zip , Reference Number , Debit/Credit Flag , SICMCC Code
14.02.2020,14.02.2020,1234,CHF0.50,"desc0","",,, "",D,
12.02.2020,13.02.2020,1234,CHF34.65,"desc1","desc2",CHE,1111, "42",D,5411
12.02.2020,13.02.2020,1234,CHF64.60,"desc3","town",CHE,1111, "42",D,5411
06.02.2020,06.02.2020,1234,-CHF2'000.50,"IHRE ZAHLUNG . BESTEN DANK","",,, "43",C,
18.01.2020,20.01.2020,1234,CHF14.00,"desc4","ZURICH",CHE,8003, "44",D,5812
14.01.2020,15.01.2020,1234,-CHF0.50,"RÜCKVERGÜTUNG RECHNUNGSGEBÜHR","",,, "45",C,
