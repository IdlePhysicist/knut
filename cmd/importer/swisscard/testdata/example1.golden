2020-01-14 "1234 RÜCKVERGÜTUNG RECHNUNGSGEBÜHR 45"
TBD                    Liabilities:CreditCard        0.5 CHF

2020-01-18 "1234 desc4 ZURICH CHE 8003 44"
Liabilities:CreditCard TBD                            14 CHF

2020-02-06 "1234 IHRE ZAHLUNG . BESTEN DANK 43"
TBD                    Liabilities:CreditCard     2000.5 CHF

2020-02-12 "1234 desc1 desc2 CHE 1111 42"
Liabilities:CreditCard TBD                         34.65 CHF

2020-02-12 "1234 desc3 town CHE 1111 42"
Liabilities:CreditCard TBD                          64.6 CHF

2020-02-14 "1234 desc0"
Liabilities:CreditCard TBD                           0.5 CHF

